swift/ql/test/library-tests/dataflow/taint/core/TaintInline.expected

```diff
@@ -1,2 +1,17 @@
-failures
 testFailures
+| conversions.swift:33:30:34:1 | // $ tainted=33\n | Missing result: tainted=33 |
+| conversions.swift:34:32:35:1 | // $ tainted=34\n | Missing result: tainted=34 |
```

Comment:
These two changes suggest something funny's going on with conversions, but not all conversions:
```
32:	sink(arg: sourceInt()) // $ tainted=32
33:	sink(arg: Int(sourceInt())) // $ tainted=33 <<< NOW MISSING
34:	sink(arg: UInt8(sourceInt())) // $ tainted=34 <<< NOW MISSING
35:	sink(arg: Float(sourceInt())) // $ tainted=35
36:	sink(arg: String(sourceInt())) // $ tainted=36
37:	sink(arg: String(sourceInt()).utf8) // $ tainted=37
```
Perhaps just conversions between integral types?  The rest of the differences in this file seem consistent with this idea.